x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_summary/monitor_alerts.tsx

```diff
@@ -145,7 +145,7 @@ export const MonitorAlerts = ({
                   { field: 'kibana.alert.status', values: ['active'] },
                   ...(locationFilter ?? []),
                 ],
-                color: theme.eui.euiColorVis7_behindText,
+                color: euiTheme.colors.vis.euiColorVis7,
```

Comment:
Blocking: We had to update our guidance on these `_behindText` colors.
We'll need the ensure color parity for Amsterdam here, which means we need to add conditional colors:

```
const isAmsterdam = euiTheme.flags.hasVisColorAdjustment // you can also check on the euiTheme.themeName !== 'EUI_THEME_BOREALIS' if you prefer

color: isAmsterdam ? euiPaletteColorBlindBehindText()[7] : euiTheme.colors.vis.euiColorVis7
```

Same applies for any further usages of `_behindText` colors further down.